charts/nginx-ingress/README.md

```diff
@@ -471,6 +471,7 @@ The following tables lists the configurable parameters of the NGINX Ingress Cont
 |`controller.readOnlyRootFilesystem` | Configure root filesystem as read-only and add volumes for temporary data. Three major releases after 3.5.x this argument will be moved permanently to the `controller.securityContext` section. | false |
 |`controller.enableSSLDynamicReload` | Enable lazy loading for SSL Certificates. | true |
 |`controller.telemetryReporting.enable` | Enable telemetry reporting. | true |
+|`controller.enableWeightChangesWithoutReload` | Enable weight changes without reloading the NGINX configuration. | false |
```

Comment:
Do we also need to call out `May require increasing variable_hash_bucket_size, and variable_hash_max_size in the ConfigMap based on the number of two-way splits` here?

Reply:
yes! thanks